src/CryptoAxus.API/Controllers/ArtistController.cs

```diff
@@ -158,6 +160,35 @@ public async Task<IActionResult> PostArtist([FromBody] CreateArtistDto artist)
         };
     }
 
+    /// <summary>
+    /// Patch Artist by UserWalletAddress
+    /// </summary>
+    /// <param name="artist"></param>
+    /// <param name="userWalletAddress"></param>
+
+    /// <response code="200">Success response with 204 code and information message about update</response>
+    /// <response code="404">Not Found response with 404 code and information message</response>
+    /// <response code="400">Bad Request response with 400 code and information message</response>
+    /// <returns></returns>
+    [HttpPatch("{userWalletAddress:required}", Name = "PatchArtist", Order = 6)]
+    [RequiresParameter(Name = "userWalletAddress", Required = true, Source = OpenApiParameterLocation.Path, Type = typeof(string))]
+    [RequiresParameter(Name = "artistDto", Required = true, Source = OpenApiParameterLocation.Body, Type = typeof(JsonPatchDocument<UpdateArtistDto>))]
+    [SwaggerRequestExample(typeof(PatchArtistRequest), typeof(PatchArtistRequestExample))]
+    [ProducesResponseType(typeof(PatchArtistResponse), (int)HttpStatusCode.OK)]
+    [ProducesResponseType(typeof(NotFoundPatchArtistResponse), (int)HttpStatusCode.NotFound)]
+    [ProducesResponseType(typeof(BadRequestPatchArtistResponse), (int)HttpStatusCode.BadRequest)]
+    public async Task<IActionResult> PatchArtist([FromRoute] string userWalletAddress, [FromBody]  JsonPatchDocument<UpdateArtistDto> artist)
```

Comment:
Extra spacing between `[FromBody]` and `JsonPatchDocument<UpdateArtistDto>`.

Reply:
Resolved